mitodl/open-discussions
# Cc/field pages v0

#### Pre-Flight checklist


#### What are the relevant tickets?
Toward https://github.com/mitodl/open-discussions/issues/3642

#### What's this PR do?
This PR adds the field pages, and creates a `<FieldPageSkeleton />` component to help create other field pages (e.g., editing fields).

See individual comments for more.

#### How should this be manually tested?
1. Make some fields. Add images if you want.
2. Check homepage is still responsive. Visit `http://cac.od.odl.local:8063/infinite` and make screen wide / narrow. For very wide pages, the content should still be centered with a max width of ~1000px.
3. Visit  http://cac.od.odl.local:8063/infinite/fields/dogs You should see a field page and tabs should work. I'll fill in more details later.

#### Screenshots (if appropriate)

<img width="1393" alt="Screen Shot 2022-07-26 at 12 41 34 PM" src="https://user-images.githubusercontent.com/9010790/181062846-87d4ec25-5407-4ac5-8ed1-736068942038.png">
